buildconfig/Setup_Win_Camera.in

```diff
@@ -0,0 +1 @@
+_camera src_c/_camera.c src_c/camera_windows.c -lMfplat -lMf -lMfuuid -lMfreadwrite -lOle32 $(SDL) $(DEBUG)
```

Comment:
This should probably be Setup_Windows.in to be consistent with the other filenames.

Reply:
There's already a Setup_Win_Common that was confusing me, and wasn't set up to parse information in this format.

Also, it's conditional on the python version, so having it as its own separate thing makes some sense.